scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -198,7 +199,7 @@ private[types] object ConverterProvider {
         case t if t =:= typeOf[Geography] =>
           q"$tree.wkt"
         case t if t =:= typeOf[Json] =>
-          q"$tree.wkt"
+          q"$tree.asJackson"
```

Comment:
According to the doc [here](https://github.com/apache/beam/blob/0cafb4644e31ada2929ab897297cc78f76fe006c/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L3755), loading json in avro expects a string type, annotated with the `JSON` sqlType property

Reply:
There is the same comment for BigQuery, but after a reverse-eng. they meant JsonNode. If opened a PR just to change the warning: https://github.com/apache/beam/pull/33121

However I can keep `asJackson` on line 417, and `.wkt` on line 202.

Reply:
c0303bbbe64b81d29d76659a10b205332631c67b